app/src/main/java/org/simple/clinic/drugs/search/DrugSearchModel.kt

```diff
@@ -2,16 +2,19 @@ package org.simple.clinic.drugs.search
 
 import android.os.Parcelable
 import kotlinx.parcelize.Parcelize
+import org.simple.clinic.drugs.selection.custom.drugfrequency.country.DrugFrequencyChoiceItem
 
 @Parcelize
 data class DrugSearchModel(
-    val searchQuery: String
+    val searchQuery: String,
+    val drugFrequencyToFrequencyChoiceItemMap: Map<DrugFrequency?, DrugFrequencyChoiceItem>?
```

Comment:
If we are using this in the UI layer only, can't we just inject it directly to the adapter or screen? WDYT? 🤔